src/sidebar/components/tag-editor.js

```diff
@@ -206,17 +208,25 @@ function TagEditor({ onEditTags, tags: tagsService, tagList }) {
           );
         })}
       </ul>
-      <input
-        list={datalistId}
-        onInput={handleOnInput}
-        onKeyPress={handleKeyPress}
-        onKeyUp={handleKeyUp}
-        ref={inputEl}
-        placeholder="Add tags..."
-        className="tag-editor__input"
-        type="text"
-      />
-      {suggestionsList()}
+      <span ref={closeWrapperRef}>
+        <input
+          onInput={handleOnInput}
+          onKeyDown={handleKeyDown}
+          onFocus={handleFocus}
+          ref={inputEl}
+          placeholder="Add new tags"
```

Comment:
I changed this value because when testing a11y it read this out with the word "ellipsis" and that sucks! I could not figure out how to get it to not read this so I figured I would just make it work for screen readers in the first place.

Reply:
I guess if you wanted to have the visual and verbal text be different you'd need to set an `aria-label` property. The solution you've gone for here seems fine to me though.

Reply:
I agree with @robertknight in terms of replacing the `placeholder` with something that doesn’t conflict with a11y issues.